pr-data.csv

```diff
@@ -3020,7 +3020,7 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedGroupUsingReferencedAttribute,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedGroupWithoutDefiningReferencedAttribute,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedUserUsingReferencedAttribute,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedUserWithoutDefiningReferencedAttribute,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedUserWithoutDefiningReferencedAttribute,ID,MovedOrRenamed,,https://github.com/apache/nifi/commit/e4ec0cb20d63a38c4e8da169b0bbd26d0317fb90
```

Comment:
The commit SHA in the note seems to be wrong.

Reply:
email with details sent